### ISSUE TITLE
error: cannot find symbol import com.pubnub.api.models.consumer.pubsub.objects.PNMembershipResult

### ISSUE THREAD

**Issue by robertmryan** [pubnub/android-quickstart-platform#1]
I'm sure it’s something basic, but I've cloned this repo, opened it in Android Studio, replaced the keys, and get this error message on [line 22 of `MainActivity.java`](https://github.com/pubnub/android-quickstart-platform/blob/master/app/src/main/java/com/pubnub/android/pnquickstart/MainActivity.java#L22):

```
/.../android-quickstart-platform/app/src/main/java/com/pubnub/android/pnquickstart/MainActivity.java:22: error: cannot find symbol
import com.pubnub.api.models.consumer.pubsub.objects.PNMembershipResult;
                                                    ^
  symbol:   class PNMembershipResult
  location: package com.pubnub.api.models.consumer.pubsub.objects
```

I'm getting a similar warning for all three of these import statements:

```java
import com.pubnub.api.models.consumer.pubsub.objects.PNMembershipResult;
import com.pubnub.api.models.consumer.pubsub.objects.PNSpaceResult;
import com.pubnub.api.models.consumer.pubsub.objects.PNUserResult;
```

Also, on [line 48](https://github.com/pubnub/android-quickstart-platform/blob/master/app/src/main/java/com/pubnub/android/pnquickstart/MainActivity.java#L48) I'm receiving:

```
/.../android-quickstart-platform/app/src/main/java/com/pubnub/android/pnquickstart/MainActivity.java:48: error: <anonymous com.pubnub.android.pnquickstart.MainActivity$1> is not abstract and does not override abstract method file(PubNub,PNFileEventResult) in SubscribeCallback
        pubnub.addListener(new SubscribeCallback() {
                                                   ^
```

I'm assuming that there is something basic that I’m overlooking. Any guidance you can offer would be greatly appreciated.

In case it’s relevant, here is my configuration:

Android Studio 4.0.1
Build #AI-193.6911.18.40.6626763, built on June 24, 2020
Runtime version: 1.8.0_242-release-1644-b3-6222593 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Registry: ide.new.welcome.screen.force=true
Non-Bundled Plugins: org.jetbrains.kotlin, com.google.services.firebase

**Comment by robertmryan** [comment]
Simple gradle sync was all I needed.

**Comment by robertmryan** [comment]
One the first issue, for some reason the gradle dependency version number of `4.+` wasn't working. But when I specified `4.32.+`, for example, it did.

**Comment by robertmryan** [comment]
I'm now seeing that 4.33.0 removed these 

```java
import com.pubnub.api.models.consumer.pubsub.objects.PNMembershipResult;		
import com.pubnub.api.models.consumer.pubsub.objects.PNSpaceResult;		
import com.pubnub.api.models.consumer.pubsub.objects.PNUserResult;		
```

Perhaps they're replaced with the following?

```java
import com.pubnub.api.models.consumer.objects_api.channel.PNChannelMetadataResult;
import com.pubnub.api.models.consumer.objects_api.membership.PNMembershipResult;
import com.pubnub.api.models.consumer.objects_api.uuid.PNUUIDMetadataResult;
```

Bottom line, this repo is not up to date with 4.33.

**Comment by robertmryan** [comment]
I fixed these issues in PR #2.
